Cargo.toml

```diff
@@ -63,31 +48,27 @@ tikv-client-store = { version = "0.2.0", path = "tikv-client-store" }
 
 [dev-dependencies]
 clap = "2"
-fail = { version = "0.4", features = [ "failpoints" ] }
-mock-tikv = {path = "mock-tikv"}
+fail = { version = "0.4", features = ["failpoints"] }
 proptest = "1"
 proptest-derive = "0.3"
 serial_test = "0.5.0"
 simple_logger = "1"
-tokio = { version = "1", features = [ "sync", "rt-multi-thread", "macros" ] }
-reqwest = {version = "0.11", default-features = false, features = ["native-tls-vendored"]}
+tokio = { version = "1", features = ["sync", "rt-multi-thread", "macros"] }
+reqwest = { version = "0.11", default-features = false, features = [
+  "native-tls-vendored",
```

Comment:
It's better to provide `native-tls` and `rustls` switch.

Reply:
We need to further discuss this aspect. Let's address it in a separate PR.